kaitai_utils.py

```diff
@@ -24,6 +25,24 @@ def get_ksy_file():
         raise ValueError("ERROR: Found ", str(len(ksy_file_list)), " .ksy files in /structs, which is not exactly 1.")
     return os.path.join(path, ksy_file_list[0])
 
+def create_bytearray_child_traces(data_binary: BinaryIO, class_type: Type[KaitaiStruct], trace: ExtractionTrace, path: str):
+    parsed_kaitai_struct = class_type.from_io(data_binary)
+    bytearrays_to_traces(parsed_kaitai_struct, trace, path)
+
+
+def bytearrays_to_traces(instance: Any, trace: ExtractionTrace, path: str):
+    parameters_dict = _parameters_dict(instance)
+    for key, value_object in parameters_dict.items():
+        if not key.startswith("_") and value_object is not None:
+            if _is_kaitai_struct(value_object):
+                bytearrays_to_traces(value_object, trace, path + '.' + key)
+            elif _is_list(value_object):
+                for value_index, value in enumerate(value_object):
+                    bytearrays_to_traces(value, trace, path + f'.[{value_index}]')
+            elif isinstance(value_object, bytes):
+                child_builder = trace.child_builder(path)
+                child_builder.update(data={'raw': value_object}).build()
```

Comment:
I have a sad realization: child_builders have limited blob-sizes (defined in `hansken_extraction_plugin.runtime.constants.MAX_CHUNK_SIZE`). Maybe protect ourselves here for larger values? A proper fix in the SDK itself is possible, but that is out of scope of this issue and probably takes a day or two to implement).

Note: this is not an issue if we store the bytes as ranges (but that is an upcoming task).

Reply:
Added if statement that makes sure we only create a child trace when the length of the byte array is smaller than this.

Reply:
I suggest that we mention this explicitly in the documentation